modules/va_forms/app/swagger/va_forms/form_model_swagger.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module VaForms
+  class FormModelSwagger
+    include Swagger::Blocks
+
+    swagger_schema :FormsIndex do
```

Comment:
Feels like this should just be the forms model since it won't change between the Index and Show

Reply:
They actually do change as the show includes the versions. I Added a new block for the show that includes versions.